extensions/@shopgate-product-reviews/frontend/RatingStars/index.jsx

```diff
@@ -12,11 +13,12 @@ const { hasReviews } = appConfig;
  * @return {JSX}
  */
 const RatingStars = ({ product }) => {
-  if (!hasReviews || !product || !product.rating || product.rating.average === 0) {
-    return null;
-  }
-
-  return <Stars value={product.rating.average} />;
+  const { showEmptyRatingStars = false } = useWidgetSettings('@shopgate/engage/rating');
```

Comment:
```suggestion
  const { showEmptyRatingStars = false } = useWidgetSettings('@shopgate/engage/rating') || {};
```
because it can return undefined if the config is not set. Please change also the other places

Reply:
nvm, useWidgetSettings returns always an empty {}